Project.toml

```diff
@@ -20,7 +20,7 @@ TileProviders = "263fe934-28e1-4ae9-998a-c2629c5fede6"
 
 [compat]
 Colors = "0.12"
-Extents = "0.1"
+Extents = "0.1.2"
```

Comment:
could you also please restrict the version for ImageMagick_jll? this is related to this issue, https://github.com/MakieOrg/Tyler.jl/issues/58, Tyler doesn't work for me without explicitly fixing the version here.